src/moscot/base/problems/_mixins.py

```diff
@@ -720,3 +720,90 @@ def compute_entropy(
         if key_added is not None:
             self.adata.obs[key_added] = df
         return df if key_added is None else None
+
+    def compute_variance(
+        self: AnalysisMixinProtocol[K, B],
+        source: K,
+        target: K,
+        forward: bool = True,
+        latent_space_selection: Union[str, list[str]] = "X_pca",
```

Comment:
let's not call it latent space. can also be raw space, e.g. gene space. Also, the types are not clear